src/generic/numberparsing.h

```diff
@@ -374,9 +374,9 @@ never_inline bool parse_large_integer(const uint8_t *const buf,
 // content and append a space before calling this function.
 //
 // Our objective is accurate parsing (ULP of 0 or 1) at high speed.
-really_inline bool parse_number(const uint8_t *const buf,
-                                const uint32_t offset,
-                                bool found_minus,
+really_inline bool parse_number(UNUSED const uint8_t *const buf,
```

Comment:
I don't mind the UNUSED qualifiers here, but is there a specific reason why they were added... is it consistent throughout our code base?

Reply:
I added them because of warnings when certain #ifdefs are passed (probably no_number_parsing or something?)